tests/integrations/flax/data_test.py

```diff
@@ -0,0 +1,6 @@
+from tango.integrations.flax import DataLoader
+
+
+def test_DataLoader() -> None:
+    assert "flax::numpy_dataloader" in DataLoader.list_available()
+    assert "flax::dataset_dataloader" in DataLoader.list_available()
```

Comment:
We should also have tests that actually read some data for the data loaders.

Reply:
Added